deal.II-toolchain/packages/gmsh.package

```diff
@@ -1,39 +1,58 @@
-# see http://gmsh.info/src/
-VERSION=3.0.6
-CHECKSUM=78bbfdd695b29cfef18a99f93226afc5
+################################################################################
+## GMSH                                                                       ##
+################################################################################
 
-#VERSION=2.16.0
-#CHECKSUM=762c10f159dab4b042e3140b1c348427
+# Option {ON|OFF}: Choose whether to load the tarball or the git repository.
+CANDI_GMSH_LOAD_TARBALL=OFF
```

Comment:
Is there a reason to go with git by default? I would think tarballs are faster to download.

Reply:
Not really... Initially I thought of getting always the newest develop-version. However, using stable version is better. I changed it and test it currently with CI on my fork. As soon it's working I will update this PR.